docs/src/main/sphinx/connector/object-storage-file-formats.rst

```diff
@@ -89,6 +89,13 @@ with Parquet files performed by supported object storage connectors:
         bloom filters by default. The equivalent catalog session property is
         ``parquet_use_bloom_filter``.
       - ``true``
+    * - ``parquet.use-column-index``
```

Comment:
Do these apply to all connectors that allow Parquet?

Reply:
Narrowed down.